requirements.txt

```
@@ -10,4 +10,5 @@ tiktoken==0.3.3
 gTTS==2.3.1
 docker
 duckduckgo-search
-google-api-python-client #(https://developers.google.com/custom-search/v1/overview) 
+google-api-python-client #(https://developers.google.com/custom-search/v1/overview)
+pinecone-client
```

Can we pin the version? 2.2.1 seems to be working
```suggestion
pinecone-client==2.2.1
```